src/Arcus.EventGrid.Testing/Infrastructure/Hosts/HybridConnectionHost.cs

```diff
@@ -17,29 +17,21 @@ public class HybridConnectionHost
         private readonly HybridConnectionListener _hybridConnectionListener;
         private readonly ILogger _logger;
 
-        private HybridConnectionHost(string hostId, HybridConnectionListener hybridConnectionListener, ILogger logger)
+        private HybridConnectionHost(HybridConnectionListener hybridConnectionListener, ILogger logger)
         {
-            Guard.NotNullOrWhitespace(hostId, nameof(hostId));
             Guard.NotNull(hybridConnectionListener, nameof(hybridConnectionListener));
             Guard.NotNull(hybridConnectionListener, nameof(hybridConnectionListener));
 
-            HostId = hostId;
-
             _logger = logger;
             _hybridConnectionListener = hybridConnectionListener;
         }
 
-        /// <summary>
-        ///     Id of the host
-        /// </summary>
-        public string HostId { get; }
-
         /// <summary>
         ///     Gets the event envelope that includes a requested event (Uses exponential back-off)
         /// </summary>
         /// <param name="eventId">Event id for requested event</param>
         /// <param name="retryCount">Amount of retries while waiting for the event to come in</param>
-        public string GetReceivedEvent(string eventId, int retryCount = 10)
+        public string GetReceivedEvent(string eventId, int retryCount = 5)
```

Comment:
Any specific reason why decrease from 10 to 5?

Reply:
Was done by accident, will revert